doc/lsst.daf.butler/queries.rst

```diff
@@ -441,23 +442,26 @@ Few examples of valid expressions using some of the constructs:
 Query result ordering
 ---------------------
 
-Few query methods (`~Registry.queryDataIds` and `~Registry.queryDimensionRecords`) support special constructs for ordering and limiting the number of the returned records. These methods return iterable objects which have ``order_by()`` and ``limit()`` methods. Methods modify the iterable object and should be used before iterating over resulting records, for convenience the methods can be chained, see example below.
+Butler query methods (`Butler.query_datasets`, `Butler.query_data_ids`, and
```

Comment:
General comment that RST preferred style is one sentence per line rather than line breaks mid sentence.